### ISSUE TITLE
spotless:apply for Java does not format unused imports correctly 

### ISSUE THREAD

**Issue by FlorianZipser** [diffplug/spotless#402]
When formatting unused imports in Java code with spotless plugin for maven, the imports are removed, but unnecessary lines remain.

Apache Maven 3.6.0
spotless version 1.23

For instance using spotless:apply to the following code

```
package foo;

import unused

public class Foo{
}
```

will produce

```
package foo;


public class Foo{
}
```

Includeing two linebreaks. This causes spotless:check to fail.

**Comment by nedtwigg** [comment]
Can you copy-paste your spotless configuration block?

**Comment by FlorianZipser** [comment]
Sure, sorry for the delay:

**Comment by nedtwigg** [comment]
As of `maven-plugin 1.31.0`, paddedcell is always enabled, which would fix this problem.  Fixed by #565
